programs/github

```diff
@@ -79,7 +82,7 @@ if flags.branch and flags.tag then
 	return printError('--branch and --tag cannot both be specified')
 end
 
-if action == 'clone' then
+if action.name == 'Clone' then
```

Comment:
I'd argue `if action == validActions.clone` is better (and that `validActions` is better named `subcommands`)